network.go

```diff
@@ -0,0 +1,36 @@
+package exp
+
+import "net"
+
+// Conatins IP
+
+type expContainsIp struct {
+	key, str string
```

Comment:
Probably naming the fields to something related to CIDR/network/subnet and IP/address would be more appropriate.